integrations/gen_integrations.py

```
@@ -22,11 +24,16 @@
     warn,
 )
 
+sys.path.insert(0, str(SELF.parent.parent / 'packaging' / 'data'))
```

Ruff (0.16.1)

[error] Undefined name `SELF`

(F821)